apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.ExecutableCommand.cs

```diff
@@ -178,6 +179,38 @@ internal async Task<long> ExecutePartitionedUpdateAsync(CancellationToken cancel
                 return await transaction.ExecuteDmlAsync(request, cancellationToken, CommandTimeout).ConfigureAwait(false);
             }
 
+            internal async Task<string> GetInstanceEndpointUriAsync()
```

Comment:
This is better off in a InstanceHostManager class or similar.

Reply:
Done!!